apps/heft/src/plugins/JestPlugin/JestPlugin.ts

```diff
@@ -21,26 +22,35 @@ export class JestPlugin implements IHeftPlugin {
     if (FileSystem.exists(path.join(heftConfiguration.buildFolder, JEST_CONFIGURATION_LOCATION))) {
       heftSession.hooks.test.tap(PLUGIN_NAME, (test: ITestStageContext) => {
         test.hooks.run.tapPromise(PLUGIN_NAME, async () => {
-          await this._runJestAsync(heftConfiguration, test.properties.watchMode, test.properties.production);
+          await this._runJestAsync(heftSession, heftConfiguration, test);
         });
       });
+
+      heftSession.hooks.clean.tap(PLUGIN_NAME, (clean: ICleanStageContext) => {
+        this._includeJestCacheWhenCleaning(heftConfiguration, clean);
```

Comment:
Should we just disable Jest's transform cache?

Reply:
@iclanton  I like that idea, but I could not figure out how to do that.  Setting [--no-cache](https://jestjs.io/docs/en/cli.html#--cache) does NOT seem to prevent the cache from being written to disk. Setting `--clearcache` causes Jest to clear the cache, then terminate immediately without running any tests.